komga-webui/src/locales/en.json

```diff
@@ -342,21 +342,21 @@
       "field_import_barcode_isbn": "ISBN barcode",
       "field_import_comicinfo_book": "Book metadata",
       "field_import_comicinfo_collections": "Collections",
-      "field_import_comicinfo_readlists": "Read lists",
+      "field_import_comicinfo_readlists": "Reading lists",
       "field_import_comicinfo_series": "Series metadata",
       "field_import_epub_book": "Book metadata",
       "field_import_epub_series": "Series metadata",
       "field_import_local_artwork": "Local artwork",
       "field_import_mylar_series": "Series metadata",
       "field_name": "Name",
-      "field_repair_extensions": "Automatically repair incorrect file extensions",
```

Comment:
Shorter.

Reply:
I don't think shorter is better here.

Reply:
I feel like you just inverted the 2 words.

Reply:
I think "repair" is more in depth, and that "erroneous" grants some clue that it is "incorrect", in error, where the error can be corrected.

The opposite is broken, in a way that needs repair.